internal/attestation/aws/snp/validator_test.go

```diff
@@ -161,7 +162,7 @@ func TestValidateSNPReport(t *testing.T) {
 			require.NoError(err)
 
 			v := awsValidator{httpsGetter: newStubHTTPSGetter(&urlResponseMatcher{}, nil), verifier: tc.verifier, validator: tc.validator}
-			err = v.validate(vtpm.AttestationDocument{InstanceInfo: infoMarshalled}, ask, ark, [64]byte(hash), config.DefaultForAWSSEVSNP(), logger.NewTest(t))
+			err = v.validate(vtpm.AttestationDocument{InstanceInfo: infoMarshalled}, ask, ark, [64]byte(hash), config.DefaultForAWSSEVSNP(), slog.New(slog.NewTextHandler(logger.TestWriter{T: t}, nil)))
```

Comment:
nit: I'd prefer to have some kind of shorthand here, as we'll need to use this quite often. Something like:
```go
func NewTest(t *testing.T) {
    return slog.New(slog.NewTextHandler(logger.TestWriter{T: t}, nil))
}
```

Then in the tests, we could simply call `logger.NewTest(t)`. Specialized cases requiring a JSON handler or passing extra options could then just instantiate the logger themselves.